Utilities/build-script-helper.py

```diff
@@ -149,7 +181,7 @@ def add_common_args(parser):
     parser.add_argument('--sanitize-all', action='store_true', help='build using every available sanitizer in sub-directories of build path')
     parser.add_argument('--no-clean', action='store_true', help='Don\'t clean the build directory prior to performing the action')
     parser.add_argument('--verbose', '-v', action='store_true', help='enable verbose output')
-    parser.add_argument('--cross-compile-host', help='cross-compile for another host instead')
+    parser.add_argument('--cross-compile-hosts', help='List of cross compile hosts targets', default=[])
```

Comment:
Is this actually a list, or is it just a string? The default value is `[]`, but I would have expected to see `nargs=*` or similar.